boards/arduino/opta/arduino_opta_stm32h747xx_m4.dts

```diff
@@ -58,17 +30,11 @@
 
 		slot0_partition: partition@80000 {
 			label = "image-0";
-			reg = <0x00080000 DT_SIZE_K(512)>;
+			reg = <0x00080000 DT_SIZE_K(384)>;
```

Comment:
512K is the correct value for the M4 flash size. Why is this change required?

Reply:
My mistake.